storage/database/leveldb_database.go

```diff
@@ -446,7 +446,7 @@ hasError:
 			currCompRead += s.LevelRead[i]
 			currCompWrite += s.LevelWrite[i]
 		}
-		db.compTimeMeter.Mark(int64(currCompTime.Seconds() - prevCompTime.Seconds()))
+		db.compTimer.Update(time.Duration(currCompTime.Nanoseconds() - prevCompTime.Nanoseconds()))
```

Comment:
Why are there `Nanoseconds()`? Couldn't it be just :
```suggestion
		db.compTimer.Update(currCompTime - prevCompTime)
```

Reply:
You're right. I did it because GoLand complained, but it seems that it was a kind of delayed response.. 😓